gradio/components/image.py

```diff
@@ -151,6 +154,7 @@ def __init__(
             label=label,
             every=every,
             show_label=show_label,
+            show_download_button=show_download_button,
```

Comment:
But this should be removed as `show_download_button` is not a parameter of the parent class `IOComponent`
```suggestion
            show_download_button=show_download_button,
```